2023-Season/src/main/java/frc/robot/Utilities/Limelight.java

```diff
@@ -5,60 +5,54 @@
 import frc.robot.Configuration.Constants;
 
 public class Limelight {
-    private NetworkTableEntry _validTargets; 
-    private NetworkTableEntry _camMode; 
-    private NetworkTableEntry _ledMode; 
-    private NetworkTableEntry _tagId; 
+    private NetworkTableEntry _validTargets;
+    private NetworkTableEntry _camMode;
+    private NetworkTableEntry _ledMode;
+    private NetworkTableEntry _tagId;
 
     public static Limelight _instance;
-    
-    public static Limelight getInstance()
-    {
-        if (_instance == null)
-        {
+
+    public static Limelight getInstance() {
+        if (_instance == null) {
             _instance = new Limelight();
         }
 
         return _instance;
     }
 
-
     public Limelight() {
-    _validTargets = NetworkTableInstance.getDefault().getTable(Constants.kTableName).getEntry(Constants.kValidTargetKey);
-    _ledMode = NetworkTableInstance.getDefault().getTable(Constants.kTableName).getEntry(Constants.kLEDModeKey);
-    _camMode = NetworkTableInstance.getDefault().getTable(Constants.kTableName).getEntry(Constants.kCamModeKey);
-    _tagId = NetworkTableInstance.getDefault().getTable(Constants.kTableName).getEntry(Constants.kTargetTagId);
-}
-
+        _validTargets = NetworkTableInstance.getDefault()
+            .getTable(Constants.kTableName)
+            .getEntry(Constants.kValidTargetKey);
 
+        _ledMode = NetworkTableInstance.getDefault().getTable(Constants.kTableName).getEntry(Constants.kLEDModeKey);
```

Comment:
We should be consistent when formatting similar code, so these should look like the above lines or vice versa.

Reply:
lol I totally forgot about changing this :)